CHANGELOG.md

```diff
@@ -21,11 +21,11 @@ The following components are considered to be internal and they are excluded fro
 - `marker_adapter`
 - `marker_error`
 
-## [Unreleased]
+## [0.3.0] - 2023-10-05
 
-This version focussed breaking changes.
+This version introduces precompiled binaries and CI templates. This version also tried to combine all breaking changes, to make `v0.3.0` a stable baseline for new additions.
```

Comment:
I suppose we still expect breaking changes, but not as big, right?

Reply:
Correct. Maybe `...to make v0.3.0 a solid baseline...` would be better phrasing